### PR TITLE
Pre-render MBFloatingActionButton Exited state correctly

### PR DESCRIPTION
**Pull request by simonziegler**

### CODE REVIEW

File: Material.Blazor/Components/FloatingActionButton/MBFloatingActionButton.razor.cs [hunk 1]
```diff
@@ -92,7 +92,8 @@ protected override async Task OnInitializedAsync()
 
             ConditionalCssClasses
                 .AddIf("mdc-fab--mini mdc-fab--touch", () => Type == MBFloatingActionButtonType.Mini)
-                .AddIf("mdc-fab--extended", () => Type == MBFloatingActionButtonType.Extended);
+                .AddIf("mdc-fab--extended", () => Type == MBFloatingActionButtonType.Extended)
```
Comment (stefanloerwald) [inline Material.Blazor/Components/FloatingActionButton/MBFloatingActionButton.razor.cs:95]: Can `Extended` and `Mini` both be used, according to spec?

File: Material.Blazor/Components/FloatingActionButton/MBFloatingActionButton.razor.cs [hunk 2]
```diff
@@ -92,7 +92,8 @@ protected override async Task OnInitializedAsync()
...
+                .AddIf("mdc-fab--exited", () => Exited);
```
Comment (stefanloerwald) [inline Material.Blazor/Components/FloatingActionButton/MBFloatingActionButton.razor.cs:96]: Maybe a demo of this feature would be nice? One that sets `Exited` on click?

File: Material.Blazor/Components/FloatingActionButton/MBFloatingActionButton.razor.cs [hunk 3]
```diff
@@ -92,7 +92,8 @@ protected override async Task OnInitializedAsync()
 
             ConditionalCssClasses
                 .AddIf("mdc-fab--mini mdc-fab--touch", () => Type == MBFloatingActionButtonType.Mini)
```
Comment (stefanloerwald) [inline Material.Blazor/Components/FloatingActionButton/MBFloatingActionButton.razor.cs:94]: is `mdc-fab-touch` never required for non-mini FABs?

File: Material.Blazor/Components/FloatingActionButton/MBFloatingActionButton.razor.cs (hunk 2 above)
Comment (simonziegler) [reply Material.Blazor/Components/FloatingActionButton/MBFloatingActionButton.razor.cs:96]: We have this demo already, but the FABs all start as visible so the FOUC doesn't show up on the website

File: Material.Blazor/Components/FloatingActionButton/MBFloatingActionButton.razor.cs (hunk 1 above)
Comment (simonziegler) [reply Material.Blazor/Components/FloatingActionButton/MBFloatingActionButton.razor.cs:95]: They're mutually exclusive.

https://github.com/material-components/material-components-web/tree/master/packages/mdc-fab#floating-action-buttons

File: Material.Blazor/Components/FloatingActionButton/MBFloatingActionButton.razor.cs (hunk 3 above)
Comment (simonziegler) [reply Material.Blazor/Components/FloatingActionButton/MBFloatingActionButton.razor.cs:94]: Only for mini. The rule Google follow is that you need a touch target if a control is less than 48px x 48px for accessibility reasons.

https://github.com/material-components/material-components-web/tree/master/packages/mdc-fab#making-fabs-accessible

File: Material.Blazor/Components/FloatingActionButton/MBFloatingActionButton.razor.cs (hunk 2 above)
Comment (stefanloerwald) [reply Material.Blazor/Components/FloatingActionButton/MBFloatingActionButton.razor.cs:96]: I had a different demo in mind. I'll quickly add it.

**Review by stefanloerwald — APPROVED** [review]

**Pull request merged**